content/_guides/catalyst.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Catalysing and De-escalation
+category: chanop
+---
+
+People often use Libera.Chat to get help with something on their computer.
+Technical support channels are one of the things we're here for!
+
+If you have ever had a technical problem, you know how overwhelming it can be.
+Sometimes people seeking help will get very frustrated and might have a temper
+or be rude. Patient and efficient help can be vital in avoiding conflict in
+these situations.
+
+This guide will outline some good practices for both operators and helpers for
+interacting with people. It will also discuss some common situations and ideal
+ways of responding.
+
+## Only be "opped up" when necessary
+
+Most channels have a select few operators who are responsible for managing the
+culture of the channel. These are the people with `+o` flags. Most clients
+display a `@` next to their nick. Libera.Chat does not have "halfops" flags
+like some other networks.
+
+We recommend that operators in channels refrain from being "opped up" unless
+it is necessary for immediate channel management activities like changing a
+locked topic or removing a problematic user. We discourage using the `+O`
```

Comment:
I find the word "discourage" here as triggering negative reaction as I consider my critique on avoiding being opped as valid and I find it coming across as "no matter your reasoning to not follow this part, you must be a horrible person if you didn't follow this order".

Reply:
you're projecting a lot of meaning on to "discourage"

Reply:
I am aware. My request would be to rephrase this in a way that doesn't include a negation. E.g.

* remove this last phrase entirely.
* we recommend giving ops the AOP template.
* this can be achieved by using the AOP template.

Reply:
Discouraging isn't the same as forbidding. We don't forbid its use. You're not breaking any rules by doing it. You'd just be using (by our philosophy) non-best practices. Sometimes that might cause discomfort at doing something less than 100% correct, but that's not on us.